src/components/SingleStock.jsx

```diff
@@ -0,0 +1,99 @@
+import React, { useEffect, useState } from 'react';
+import { Link, useParams } from 'react-router-dom';
+import { useDispatch, useSelector } from 'react-redux';
+import { getStockDetail } from '../redux/stocks/stocksSlice';
+import '../css/SingleStock.css';
+import demologo from '../assets/demologo2.png';
+
+const SingleStock = () => {
+  const { stockDetail } = useSelector((store) => store.stocks);
+  const { loadingDetail } = useSelector((store) => store.stocks);
+  const [imgLoadError, setImgLoadError] = useState(false);
+
+  const handleImgLoad = () => {
+    console.log('This is printing');
+    setImgLoadError(true);
+    console.log('imgLoadError: ', imgLoadError);
+  };
+
+  const { symbol } = useParams();
+  console.log('symbol: ', symbol);
+
+  const singleStock = stockDetail;
+
+  const dispatch = useDispatch();
+  useEffect(() => {
+    dispatch(getStockDetail(symbol));
+  }, [dispatch]);
+
+  if (loadingDetail) {
+    return (
+      <div data-testid="stockDetail">
+        Loading...please wait
+        <Link to="/">
+          <button type="button">Go back</button>
+        </Link>
+      </div>
+    );
+  }
+
+  return (
+    <div className="single-stock" data-testid="stockDetail">
+      <div className="overhead">
+        <img
+          src={(imgLoadError)
+            ? demologo : singleStock.image}
+          alt="logo"
+          className="logo"
+          onError={handleImgLoad}
+        />
+        <p className="overhead-symbol">
+          &copy;
+          {symbol}
+        </p>
+      </div>
+      <div className="info-container">
+        <div>
+          <p>Company Name</p>
+          <p>{singleStock.companyName}</p>
+        </div>
+        <div>
+          <p>Market Cap</p>
+          <p>{singleStock.mktCap}</p>
+        </div>
+        <div>
+          <p>Current Price</p>
+          <p>{singleStock.price}</p>
+        </div>
+        <div>
+          <p>Average Volume</p>
+          <p>{singleStock.volAvg ? singleStock.volAvg : 'N/A'}</p>
+        </div>
+        <div>
+          <p>Sector</p>
+          <p>{singleStock.sector ? singleStock.sector : 'N/A'}</p>
+        </div>
+        <div>
+          <p>CEO</p>
+          <p>
+            {singleStock.ceo ? singleStock.ceo : 'N/A'}
+          </p>
+        </div>
+        <div>
+          <p>Official Site</p>
+          <p>
+            {
+            singleStock.website ? (
+              <a href={singleStock.website} target="_blank" rel="noreferrer">
+                {singleStock.website}
+              </a>
+            ) : 'N/A'
+            }
+          </p>
+        </div>
+      </div>
```

Comment:
👍 You've provided extensive information about the company in the `info-container` section. To align your project more closely with the design, consider the following:

- [ ]  In the design, there's a section title bar missing in your current implementation. To adhere to the design specifications, add a section title at the top of the `info-container` section. This title will provide context and structure to the information presented below. Moreover, After adding the title bar, ensure that the color scheme for this `info-container` section matches the design as it has used two colors for better contrast. You can use the designated colors to create a visually cohesive and aesthetically pleasing layout.

| Your | Design |
| --- | --- |
|  ![image](https://github.com/AliRezaBmeDu/metrics-webapp/assets/55840999/bfeb24a9-750a-4ba7-9d98-f3651dddff4d)| ![image](https://github.com/AliRezaBmeDu/metrics-webapp/assets/55840999/216f0f80-1bf0-4443-b595-c0787ae39e03)|

By incorporating these adjustments, you'll enhance the `info-container` section to better match the design and improve the overall user experience. 🎨📝

Reply:
done